libs/api/domains/cms/src/lib/models/organizationPage.model.ts

```diff
@@ -45,11 +55,16 @@ export const mapOrganizationPage = ({
   title: fields.title ?? '',
   slug: fields.slug ?? '',
   description: fields.description ?? '',
+  theme: fields.theme ?? 'default',
+  themeProperties: fields.themeProperties
+    ? JSON.stringify(fields.themeProperties)
+    : '',
```

Comment:
We should avoid using JSON if able since it breaks type safety between the api and web.
It seems like `themeProperties` contains a set of known properties:
```
gradientStartColor: string
gradientEndColor: string
```
Can we move this to its own model with a mapper?

Reply:
Good idea. See [a35c446](https://github.com/island-is/island.is/pull/3391/commits/a35c446825a4436f6397d3e860c2d248622618c3)